Description: Add copyright for notification-disabled-symbolic.svg #1545
Author: David Mohammed <fossfreedom@ubuntu.com>
Commit: https://github.com/UbuntuBudgie/budgie-desktop/commit/49d49ed11d06aae544fe9f1d99dab000170dda1c
Last Update: 2018-08-29

---
 data/icons/README | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/data/icons/README b/data/icons/README
index 4fa686b2..4913421a 100644
--- a/data/icons/README
+++ b/data/icons/README
@@ -3,11 +3,15 @@ icon found on 1001freedownloads, under the CC0 1.0 license

 http://www.1001freedownloads.com/free-clipart/bell-icon

+The notification-disabled-symbolic.svg file is a xfce4-notifyd
+icon under the GPL-2+ license
+copyright © 2008 Brian Tarricone <bjt23@cornell.edu>
+
 The pane-show-symbolic.svg and pane-hide-symbolic.svg icons are
 created by horst3180 and are licensed under the GPL-2.0

 Budgie Logo: CC-BY-SA-4.0. Original work by Yuri Henrique

-The remaining icons in status/ and apps/ are copyright © 2016-2017 Sam Sam Hewitt
+The remaining icons in status/ and apps/ are copyright © 2016-2017 Sam Hewitt
 https://github.com/snwh/icons
 They are available under the terms of the CC-BY-SA-4.0 license
--
2.17.1

